### PR TITLE
Merge please.

### PR DESCRIPTION
**Pull request by tyacker**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/tyler.json [hunk 1]
```diff
@@ -1,14 +1,10 @@
 {
+    "description": "My personal Website",
     "owner": {
-        "username": "G2C-DC",
-        "email": "tylercalumacker@gmail.com"
+        "username": "tylerdotnet",
```
Comment (Stef-00012) [inline domains/tyler.json:4]: the username field should be your github username

File: domains/domains/_github-pages-challenge-tylerdotnet.tyler.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "tylerdotnet",
```
Comment (Stef-00012) [inline domains/domains/_github-pages-challenge-tylerdotnet.tyler.json:3]: the username field should be your github username

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/domains/_github-pages-challenge-tylerdotnet.tyler.json (hunk 2 above)
Comment (21Z) [inline domains/domains/_github-pages-challenge-tylerdotnet.tyler.json:3]:
```suggestion
        "username": "tylercalumacker",
```

File: domains/tyler.json (hunk 1 above)
Comment (21Z) [inline domains/tyler.json:4]:
```suggestion
        "username": "tylercalumacker",
```

**Review by 21Z — CHANGES_REQUESTED** [review]

**Pull request closed without merging**